src/HttpServer.php

```diff
@@ -8,6 +8,7 @@
 use React\Http\Io\IniUtil;
 use React\Http\Io\MiddlewareRunner;
 use React\Http\Io\StreamingServer;
+use React\Http\Middleware\IdleConnectionTimeoutMiddleware;
```

Comment:
Seeing a future PR here? :eyes:

Reply:
Nothing to see here move along!

Reply:
It's up at https://github.com/reactphp/http/pull/423